integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/noarkexchange/IntegrasjonspunktImpl.java

```diff
@@ -63,9 +63,15 @@ public GetCanReceiveMessageResponseType getCanReceiveMessage(@WebParam(name = "G
         GetCanReceiveMessageResponseType response = new GetCanReceiveMessageResponseType();
         boolean certificateAvailable;
 
-        certificateAvailable = adresseRegister.hasAdresseregisterCertificate(organisasjonsnummer);
+        final ServiceRecord serviceRecord;
+        try {
+            serviceRecord = serviceRegistryLookup.getServiceRecord(organisasjonsnummer);
+        } catch (Exception e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1166)